scripts/launcher/run_one_test.py

```diff
@@ -135,6 +135,15 @@ def _create_virtual_conf(self, log_path) -> VirtualConfiguration:
 
         kernel_args = self._shell.kernel_args()
 
+        timeout_string = self._shell.get_timeout()
+        if timeout_string:
+            # looks like the test has a custom timeout set,
+            # try to parse it to an integer
+            timeout = int(timeout_string)
+        else:
+            # no custom timeout, fall back to the default 60 minute test timeout
+            timeout = 60
```

Comment:
The default value can be provided by `functions.sh`.

Reply:
Nice idea, let's do that. :)